File: ws/gavilanez/u1/ws08CDforJava/ChickenFarmSimulator2/src/ec/edu/espe/farmsimulator/view/FarmSimulator.java

package ec.edu.espe.farmsimulator.view;

import ec.edu.espe.farmsimulator.model.Chicken;
import java.util.Date;
/**
 *
 * @author Kenny Gavilanez, The Javas, DCCO-ESPE
 */
public class FarmSimulator {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        System.out.println("Kenny's Chicken Farm Simulator V 0.5");
        Chicken chicken = new Chicken(1, "Lucy", "White and brown",0, true, new Date());
        System.out.println("Chicken ---> \n" + chicken);
        
        int id=2;
        String name="Maruja";
        String color="black";
        int age=1;
        boolean molting = false;
        Date bornOnDate = new Date();
        
        chicken = new Chicken(id, name, color, age, molting, bornOnDate);
        System.out.println("Chicken 2 ---> \n" + chicken);
    }
    
}

File: hw/gualotuna/u1/HW09CDToJavaScrip/ChickenFarmSimulator2/farmsimulator.js
const Chicken = require("./Chicken");
function main() {
    console.log("Brayan's Chicken Farm Simulator V 0.5");
    
    let chicken = new Chicken(1, "Lucy", "white and brown", 0, true, new Date());
    console.log("Chicken --> \n" + chicken.toString());
    
    let id = 2;
    let name = "Maruja";
    let color = "black";
    let age = 1;
    let molting = false;
    let bornOnDate = new Date();
    
    chicken = new Chicken(id, name, color, age, molting, bornOnDate);
    console.log("Chicken 2 --> \n" + chicken.toString());
}

main();